frontend/app/component/form/candidates_votes_form/CandidatesVotesForm.tsx

```diff
@@ -146,6 +146,7 @@ export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
                 id={`candidate_votes[${candidate.number - 1}].votes`}
                 title={`${candidate.last_name}, ${candidate.initials} (${candidate.first_name})`}
                 errorsAndWarnings={isSaved ? errorsAndWarnings : undefined}
+                warningsAccepted={getIgnoreWarnings()}
```

Comment:
It's good we're consistent about the CandidatesVotesForm supporting warnings, even though the backend currently does not have any warnings for this form implemented.

However, if we support the functionality in the frontend, we should have some tests for it. The alternative would be to not have the warnings functionality for this form and throw an error if one does show up in the response from the backend.

Reply:
Added in cdd913115d1429b3f1c71bbd65829f67d5013f67